### PR TITLE
Add Deployment tools section

### PR DESCRIPTION
**Pull request by ntk148v**
Actually the first one is mine :rofl:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -101,6 +102,14 @@ Prometheus is an open-source systems monitoring and alerting toolkit.
 - [PromQL tutorial for beginners](https://medium.com/@valyala/promql-tutorial-for-beginners-9ab455142085) - PromQL tutorial for beginners by Aliaksandr Valialkin.
 - [Prometheus storage: technical terms explained](https://medium.com/@valyala/prometheus-storage-technical-terms-for-humans-4ab4de6c3d48) - Prometheus storage: technical terms for humans by Aliaksandr Valialkin.
 
+## Deployment tools
+
+- [Ansitheus - Ansible playbook to containerize, configure and deploy Prometheus ecosystem](https://github.com/ntk148v/ansitheus)
```
Comment (roaldnefs) [inline README.md]:
```suggestion
- [Ansitheus](https://github.com/ntk148v/ansitheus) - Ansible playbook to containerize, configure and deploy Prometheus ecosystem _by ntk148v_
```

File: README.md [hunk 2]
```diff
@@ -101,6 +102,14 @@ Prometheus is an open-source systems monitoring and alerting toolkit.
...
+- [Ernestas Poskus's Ansible playbook for installing Prometheus monitoring system, exporters such as: node, snmp, blackbox, thus alert manager and push gateway](https://github.com/ernestas-poskus/ansible-prometheus)
+- [Cloudalchemy's ansible-prometheus](https://github.com/cloudalchemy/ansible-prometheus)
+- [Google Cloud Platform click-to-deploy Prometheus stack](https://github.com/GoogleCloudPlatform/click-to-deploy/tree/master/k8s/prometheus)
+- [Prometheus Operator](https://github.com/coreos/prometheus-operator)
```
Comment (roaldnefs) [inline README.md]:
```suggestion
- [Prometheus Operator](https://github.com/coreos/prometheus-operator) - Prometheus Operator creates/configures/manages Prometheus clusters atop Kubernetes _by CoreOS_
```

**Review by roaldnefs — CHANGES_REQUESTED** [review]
@ntk148v, thanks for your contribution! Would you be willing to update the deployment tools entries according to my example suggestions?

**Comment by ntk148v** [comment]
@roaldnefs done :100:

**Review by roaldnefs — APPROVED** [review]
LGTM!

**Pull request merged**